backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -58,6 +82,18 @@ public void setup() {
     @DisplayName("the GET request should return all the games from the Db")
     public void getAllGamesFromDbTest() {
         //GIVEN
+        Player player1 = Player.builder().name("Mario").color("red").score(123)
+                .build();
+        Player player2 = Player.builder().name("Sanne").color("blue").score(122)
+                .build();
+        Player player3 = Player.builder().name("samBAM").color("yellow").score(100)
+                .build();
+        Player player4 = Player.builder().name("Mario").color("red").score(90)
+                .build();
+        Player player5 = Player.builder().name("Sanne").color("blue").score(80)
+                .build();
+        Player player6 = Player.builder().name("samBAM").color("yellow").score(70)
+                .build();
```

Comment:
why are you repeating player set up if you have included it in your `@BeforeEach` method